### PR TITLE
added break and continue documentation

### LINKED ISSUE #1995: Java 8.1: Break and continue
### Description

Description Add audio, Documentation, and Video regarding the **Break and continue**.

____

### Location of File to be added

The files should be added inside the `Break and continue`  folder which is inside the `Jump Statements`  folder present in the   `Java`  main folder.  The documentation file can be in the `.md`/`.py`/`.ipynb` format. Audio should be in `.mp3` format and to be submitted through a Google drive link. Video can be in any format and should be submitted through a Google Drive link.

 **Note:**
 - You are required to comment with `\assign CONTENT_TYPE` , so that other content types can be created at the same time.
 - Failing to comment in the above manner will lead to the removal of assignment from the issue even if the bot assigns.
 - You are required to do a PR within 7 days otherwise you will be unassigned and others will be assigned.
 - Issues will be assigned on a first come first serve basis.

### Domain

Java_Domain

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by rajmadhu0406**
<hr>

## Description 📜

I have added documentation about break and continue statement. The documentation contains syntax and examples along with written programs. I have also included a .gif file for better understandability.
Fixes #1995   

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [x] Java_Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by garvitraj** [comment]
@AkMo3 pls have a final look

File: Java/Jump Statements/Break and continue/Break and Continue.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+```sh
+Syntax : break;
+```
+
+<br>
+
+> -As the name suggests, __"break"__ statement is used to break out of a loop. <br>
+-It is used to modify the flow of code. <br>
+-We generally use __"break"__ statements in loops and switch case <br>
+-Wherever the __"break"__ keyword is used, the loop is instantly terminated. <br>
```
Comment (AkMo3) [inline Java/Jump Statements/Break and continue/Break and Continue.md:16]: Hi, just a minor change. Please change it to the bulletin. Like `1,` or circle

**Review by AkMo3 — CHANGES_REQUESTED** [review]

**Review by garvitraj — APPROVED** [review]

**Review by AkMo3 — APPROVED** [review]

File: Java/Jump Statements/Break and continue/Break and Continue.md (hunk 1 above)
Comment (AkMo3) [reply Java/Jump Statements/Break and continue/Break and Continue.md]: > Hi, just a minor change. Please change it to the bulletin. Like 1, or circle and all other similar places.
> indicates you are referring to something said in the past.

Done by me

**Pull request merged**